dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java

```diff
@@ -114,4 +115,21 @@ private static void startDatabase() throws IOException, SQLException {
         final ScriptRunner runner = new ScriptRunner(ds.getConnection(), true, true);
         runner.runScript(new FileReader("sql/dolphinscheduler_h2.sql"));
     }
+
+    public static String getCurrentPath(Class<?> cls) {
+        String path = cls.getProtectionDomain().getCodeSource().getLocation().getPath();
+        path = path.replaceFirst("file:/", "");
+        path = path.replaceAll("!/", "");
```

Comment:
I suppose there should not be `!/` in the path, because this path is only use when running in IDE, if it's run in distribution package, like using command line, it will fall back to the configuration in `registry.properties`

Reply:
> I suppose there should not be `!/` in the path, because this path is only use when running in IDE, if it's run in distribution package, like using command line, it will fall back to the configuration in `registry.properties`

Do you mean that remove `path = path.replaceAll("!/", "");` ?

Reply:
Yes @QuakeWang !

Reply:
I have updated it : )